ykcompile/src/lib.rs

```diff
@@ -1570,7 +1571,7 @@ mod tests {
         inputs.2 = inputs.0 + 2;
         inputs.3 = inputs.0 + inputs.0;
         let sir_trace = th.stop_tracing().unwrap();
-        let tir_trace = TirTrace::new(&*sir_trace).unwrap();
+        let tir_trace = TirTrace::new(&*SIR, &*sir_trace).unwrap();
```

Comment:
I've wondered for a while if there's a way to get rid of all of this `&*` malarky. It's because we get a `Box` but we need a reference, and they don't coerce...

Reply:
It is not really a `Box`, but instead a type generated by `lazy_static` that in the deref impl initializes the static if necessary.